src/tutorial/runtime/runtime-configuration.rst

```diff
@@ -107,12 +106,7 @@ Job Submission
            script = big-executable
 
            # Submit to the host "big-computer".
-           [[[remote]]]
-               host = big-computer
-
-           # Submit the job using the "slurm" job runner.
-           [[[job]]]
-               batch system = slurm
+           platform = big_computer_platform_with_job_runner
 
```

Comment:
As above, maybe give a global.cylc snippet showing the `big_computer_platform_with_job_runner` configured to use slurm